entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+STORE=$(realpath /mddb/"${STORE:-cs.repo}")
+MDDB="/mddb/${MDDB:-metadata.db}"
+RECIPES="${RECIPES:-/recipes/}"
```

Comment:
Maybe copy some example recipes from bdcs-cli ?

Reply:
Yeah, that's a good idea